packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleRadioButtonTests.cs

```diff
@@ -0,0 +1,22 @@
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleRadioButton"/>
+/// </summary>
+public class NimbleRadioButtonTests
+{
+    [Fact]
+    public void NimbleRadioButton_Rendered_HasRadioButtonMarkup()
```

Comment:
It's fairly straightforward to write tests to validate all of the properties are getting set properly in the DOM:

See the [Combobox tests](https://github.com/ni/nimble/blob/main/packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleComboboxTests.cs) as an example.

Reply:
Added tests